src/CommunityToolkit.Maui.Core/Essentials/Badge/IBadge.shared.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CommunityToolkit.Maui.ApplicationModel;
+
+/// <summary>
+/// Badge
+/// </summary>
+public interface IBadge
```

Comment:
Let's also include a `int GetCount()` (or `int Count { get; }`).

Looking at the sample, after we add `Badge.GetCount()`, we can remove the `int count` field:

```cs
	[RelayCommand]
	void Decrement()
	{
		if (count > 0)
		{
			count--;
			badge.SetCount(count);
		}
	}
```

Becomes

```cs
	[RelayCommand]
	void Decrement()
	{
		if (badge.GetCount() > 0)
		{
			badge.SetCount(count);
		}
	}
```

Reply:
we don't know the current count (at least on Android). For android we'll return value 0.